test/e2e/storage/in_tree_volumes.go

```diff
@@ -56,6 +56,7 @@ var testSuites = []func() testsuites.TestSuite{
 	testsuites.InitSubPathTestSuite,
 	testsuites.InitProvisioningTestSuite,
 	testsuites.InitMultiVolumeTestSuite,
+	testsuites.InitVolumeExpandTestSuite,
```

Comment:
Can you add this here too: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/external/external.go#L48